ui/src/dashboards/containers/DashboardPage.js

```diff
@@ -0,0 +1,112 @@
+import React, {PropTypes} from 'react';
+import ReactTooltip from 'react-tooltip';
+
+import LayoutRenderer from 'shared/components/LayoutRenderer';
+import TimeRangeDropdown from '../../shared/components/TimeRangeDropdown';
+import timeRanges from 'hson!../../shared/data/timeRanges.hson';
+
+import {getDashboard} from '../apis';
+import {getSource} from 'shared/apis';
+
+const DashboardPage = React.createClass({
+  propTypes: {
+    params: PropTypes.shape({
+      sourceID: PropTypes.string.isRequired,
+      dashboardID: PropTypes.string.isRequired,
+    }).isRequired,
+  },
+
+  getInitialState() {
+    const fifteenMinutesIndex = 1;
+
+    return {
+      timeRange: timeRanges[fifteenMinutesIndex],
+    };
+  },
+
+  componentDidMount() {
+    getDashboard(this.props.params.dashboardID).then((resp) => {
+      getSource(this.props.params.sourceID).then(({data: source}) => {
+        this.setState({
+          dashboard: resp.data,
+          source,
+        });
+      });
+    });
+  },
+
+  renderDashboard(dashboard) {
+    const autoRefreshMs = 15000;
+    const {timeRange} = this.state;
+    const {source} = this.state;
+
+    const cellWidth = 4;
+    const cellHeight = 4;
+
+    const cells = dashboard.cells.map((cell, i) => {
+      const dashboardCell = Object.assign(cell, {
+        w: cellWidth,
+        h: cellHeight,
+        queries: cell.queries,
+        i: i.toString(),
+      });
+
+      dashboardCell.queries.forEach((q) => {
+        q.text = q.query;
+        q.database = source.telegraf;
+      });
+      return dashboardCell;
+    });
+
+    return (
+      <LayoutRenderer
+        timeRange={timeRange}
+        cells={cells}
+        autoRefreshMs={autoRefreshMs}
+        source={source.links.proxy}
+        host={this.props.params.sourceID}
```

Comment:
I think this `host` may be a problem.
I created a dashboard like this
```sh
curl -X POST -H "Content-Type: application/json" -d '{"id":0,"cells":[{"x":0,"y":0,"w":0,"h":0,"queries":[{"label":"%","query":"SELECT mean(\"usage_user\") AS \"usage_user\" FROM \"cpu\"","wheres":[],"groupbys":[]}],"type":"line"}],"name":"dashboard name"}
' "http://localhost:8888/chronograf/v1/dashboards"
```

The query that is sent to influx is
```sql
"SELECT mean("usage_user") AS "usage_user" FROM "cpu" where time > now() - 15m and "host" = '4' group by time(1m)"
```
... and the graph doesn't have any data.  I think it is because of the "host" = '4'